packages/components/Menu/src/MenuTrigger/getRevisedProps.ts

```diff
@@ -0,0 +1,48 @@
+import { AccessibilityActionEvent } from 'react-native';
+import { memoize } from '@fluentui-react-native/framework';
+import { InteractionEvent, isMouseEvent } from '@fluentui-react-native/interactive-hooks';
+import { MenuTriggerChildProps, MenuTriggerState } from './MenuTrigger.types';
+
+export const getRevisedProps = memoize(getRevisedPropsWorker);
+function getRevisedPropsWorker(state: MenuTriggerState, props: any): MenuTriggerChildProps {
+  const revisedProps = { ...state.props };
```

Comment:
This is done so that revisedProps.Foo and state.props.Foo are different references, so that calling state.props.Foo inside revisedProps.Foo doesn't re-call revisedProps.Foo. Before I had revisedProps.Foo re-call revisedProps.Foo, which resulted in an infinite recursion loop